app/assets/javascripts/votes.js

```diff
@@ -10,13 +10,13 @@ $(document).ready(function() {
           $('.update-rating-' + data[0].id).html(
             data[1]
           )} else {
-            alert('You need to sign in to vote!')
+            alert('Please sign in or create an account to vote!')
```

Comment:
Mixed double and single quotes.<br>Missing semicolon.<br>'alert' is not defined.